### PR TITLE
Test_2

### PR DESCRIPTION
**Pull request by Major666RUS**

### CODE REVIEW

File: src/team.py [hunk 1]
```diff
@@ -47,10 +47,6 @@ def get_rating(self):
         first11 = self.ideal_squad()
         return sum(pl.rating for pl in first11)
 
-    def first11_presentation(self):
```
Comment (iibykov) [inline src/team.py:50]: Dirty job, needs to be redone

**Review by iibykov — CHANGES_REQUESTED** [review]
Ok

File: src/team.py (hunk 1 above)
Comment (Major666RUS) [reply src/team.py:50]: Firstly I'm greeting you!

**Comment by dmitriy-bel** [comment]
Great job! We can accept this pull request

**Pull request merged**